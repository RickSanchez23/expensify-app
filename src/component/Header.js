import React from "react";
import { NavLink } from "react-router-dom";

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <div>
      <NavLink to="/" activeClassName="active-nav-link" exact>
        Dashboard
      </NavLink>
      <NavLink to="/create" activeClassName="active-nav-link">
        Create
      </NavLink>
      <NavLink to="/help" activeClassName="active-nav-link">
        Help
      </NavLink>
    </div>
  </header>
);

export default Header;
