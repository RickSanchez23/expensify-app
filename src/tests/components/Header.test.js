import React from "react";
import Header from "../../component/Header";
import { shallow } from "enzyme";

test("should Header render correctly", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
