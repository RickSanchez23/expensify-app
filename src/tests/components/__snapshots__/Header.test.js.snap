// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should Header render correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <div>
    <NavLink
      activeClassName="active-nav-link"
      aria-current="page"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
    <NavLink
      activeClassName="active-nav-link"
      aria-current="page"
      to="/create"
    >
      Create
    </NavLink>
    <NavLink
      activeClassName="active-nav-link"
      aria-current="page"
      to="/help"
    >
      Help
    </NavLink>
  </div>
</header>
`;
